receiver/otlpreceiver/otlp.go

```diff
@@ -162,7 +162,10 @@ func (r *otlpReceiver) startHTTPServer(ctx context.Context, host component.Host)
 	}
 
 	var err error
-	if r.serverHTTP, err = r.cfg.HTTP.ToServer(ctx, host, r.settings.TelemetrySettings, httpMux, confighttp.WithErrorHandler(errorHandler)); err != nil {
+	if r.serverHTTP, err = r.cfg.HTTP.ToServer(ctx, host, r.settings.TelemetrySettings, httpMux,
+		confighttp.WithErrorHandler(errorHandler),
+		confighttp.WithSpanFormatter(confighttp.PrefixFormatter(r.settings.ID.String())),
```

Comment:
Is this a breaking change? If this is merged will the spans that the otlpreceiver creates about itself have new names?

Reply:
@TylerHelmuth Is your concern that we should not do this if it is a breaking change?